### PR TITLE
add new pcb - chlx piche

### PR DESCRIPTION
**Pull request by gaclee3b**
## Description

Adding new Alice-style 60% pcb variant capable of being used in split / tented applications.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/chlx/piche60/readme.md
Comment (waffle87) [inline keyboards/chlx/piche60/readme.md:3]: Please fill this out.

File: keyboards/chlx/piche60/readme.md
Comment (waffle87) [inline keyboards/chlx/piche60/readme.md:3]:
```suggestion
![piche60 PCB](https://i.imgur.com/CBeJSUy.jpeg)
```

**Review by waffle87 — APPROVED** [review]

File: keyboards/chlx/piche60/keymaps/via/config.h
Comment (zvecr) [inline keyboards/chlx/piche60/keymaps/via/config.h:1]: Please remove this file, it builds fine with the default setting of 4.

File: keyboards/chlx/piche60/keyboard.json
Comment (zvecr) [inline keyboards/chlx/piche60/keyboard.json:5]: Bootmagic is not configured to use top left key
```suggestion
    "bootloader": "atmel-dfu",
    "bootmagic": {
        "matrix": [1, 1]
    },
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by gaclee3b** [comment]
can someone help get this approved and merged?

File: keyboards/chlx/piche60/readme.md
Comment (drashna) [inline keyboards/chlx/piche60/readme.md:12]: No link to a store page, or repo for pcb fabrication?

Comment (gaclee3b) [reply keyboards/chlx/piche60/readme.md:12]: I usually just run orders through r/mk bulk sale posts and google forms, but haven't run one yet for this. do I just link that later or are you asking for an official store of sorts before the merge is approved?

**Comment by gaclee3b** [comment]
please don't stale this. I'm still unsure what change is being requested here. I am under the impression I fulfilled all PR requirements and have already gotten one approval.

**Comment by zvecr** [comment]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

File: keyboards/chlx/piche60/readme.md
Comment (drashna) [reply keyboards/chlx/piche60/readme.md:12]: A mention of that should be fine, I think.

**Review by drashna — APPROVED** [review]

**Pull request merged**
